geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/StatParser.scala

```diff
@@ -102,6 +102,13 @@ private class StatParser extends BasicParser {
     }
   }
 
+  private def stats: Rule1[Stat] = rule {
+    "Stats(" ~ string ~ ")" ~~> { attributes =>
```

Comment:
can we change this to 'DescriptiveStats', and change the associated rule name? 'stats' is pretty generic, since they're all stats...